ktor-network/ktor-network-tls/jvm/test/io/ktor/network/tls/tests/TLSConfigBuilderTest.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.network.tls.tests
+
+import io.ktor.network.tls.*
+import io.ktor.network.tls.certificates.*
+import io.ktor.network.tls.extensions.*
+import kotlin.test.*
+
+internal class TLSConfigBuilderTest {
+    @Test
+    fun specificAliasInKeyStore() {
```

Comment:
Can you split this test into two? With fixed alias and without it.

Reply:
Of course :D